content/en/docs/tasks/security/authorization/authz-dry-run/index.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Dry Run
+description: Shows how to dry-run an authorization policy without enforcing it.
+weight: 65
+keywords: [security,access-control,rbac,authorization,dry-run]
+owner: istio/wg-security-maintainers
+test: no
+---
+
+This task shows you how to set up an Istio authorization policy using a new [experimental annotation `istio.io/dry-run`](/docs/reference/config/annotations/)
+to dry-run the policy without actually enforcing it.
+
+The dry-run annotation allows you to better understand the effect of an authorization policy before applying it to
+the production traffic. This helps to reduce the risk of breaking the production traffic caused by incorrect or wrong
+authorization policy.
+
+{{< boilerplate experimental-feature-warning >}}
+
+## Before you begin
+
+Before you begin this task, do the following:
+
+* Read the [Istio authorization concepts](/docs/concepts/security/#authorization).
+
+* Follow the [Istio installation guide](/docs/setup/install/istioctl/) to install Istio.
+
+* Deploy Zipkin for checking dry-run tracing results. Follow the [Zipkin task](/docs/tasks/observability/distributed-tracing/zipkin/)
+  to install Zipkin in the cluster. Make sure the sampling rate is set to 100 that allows you to quickly reproduce the trace span in the task.
+
+* Deploy Prometheus for checking dry-run metric results. Follow [Prometheus task](/docs/tasks/observability/metrics/querying-metrics/)
+  to install the Prometheus in the cluster.
+
+* Deploy test workloads:
+
+    This task uses two workloads, `httpbin` and `sleep`, both deployed in namespace `foo`.
+    Both workloads run with an Envoy proxy sidecar. Deploy the `foo` namespace and workloads with the following command:
+
+    {{< text bash >}}
+    $ kubectl create ns foo
+    $ kubectl label ns foo istio-injection=enabled
+    $ kubectl apply -f @samples/httpbin/httpbin.yaml@ -n foo
+    $ kubectl apply -f @samples/sleep/sleep.yaml@ -n foo
+    {{< /text >}}
+
+* Enable proxy debug level log for checking dry-run logging results:
+
+    {{< text bash >}}
+    $ istioctl pc log deploy/httpbin.foo --level "rbac:debug" | grep rbac
```

Comment:
I think we usually expand all subcommands.

Reply:
updated to `proxy-config`.